microsoft-edge/devtools-guide-chromium/issues/index.md

```diff
@@ -43,42 +43,57 @@ Feedback in the **Issues** tool is provided by several sources, including the Ch
 
 1. Go to a webpage that contains issues to fix.  For example, open the [accessibility-testing demo page](https://microsoftedge.github.io/Demos/devtools-a11y-testing/) in a new tab or window.
 
-1. Open DevTools.  After a few seconds, the **Issues counter** (![Issues counter](./index-images/issues-counter-icon.png)) appears in the upper right corner of DevTools.
+1. Right-click the webpage and then select **Inspect**.  DevTools opens.
+
+<!--
+   After a few seconds, the **Issues counter** (![Issues counter](./index-images/issues-counter-icon.png)) appears in the upper right corner of DevTools.
 
 1. Refresh the page, because some issues are reported based on network requests.  Notice the updated count in the **Issues counter**.
+-->
+
+1. In the Quick View toolbar at the bottom of DevTools, select the **Issues** tab.
+
+   If Quick View isn't displayed, select **Customize and control DevTools** (**...**), and then select **Toggle Quick View panel** (**Esc**).  If the **Quick View** toolbar doesn't have the **Issues** tab, in the **Quick View** toolbar, click the **More tools** (**+**) button, and then select **Issues**.
 
-1. Select the **Issues counter**.  The **Issues** tool opens with issues grouped into different categories.
+   The **Issues** tool groups issues into categories, such as **Accessibility**, **Performance**, **Security**, and **Other**:
 
    ![Categories of issues in the Issues tool on the demo page](./index-images/categories.png)
 
 
 ### Other ways to open the Issues tool
 
-There are several additional ways to open the **Issues** tool:
-*  Click the **More Tools** (**+**) menu in the main panel or the **Drawer**, and then select **Issues**.
+There are multiple ways to open the **Issues** tool, such as:
+
+*  In the **Activity Bar** or in **Quick View**, click the **More tools** (**+**) button, and then select **Issues**.
+*  In the **Elements** tool, in the DOM tree, find a wavy-underlined element name, and then press and hold **Shift** and click the element.  Or, right-click a wavy-underlined element, and then select **View issues**.
+<!-- legacy ui
 *  Select **Customize and control DevTools** > **More tools** > **Issues**.
-*  In the DOM tree in the **Elements** tool, press **Shift** and then click a wavy-underlined element name.  Or, open the context menu on a wavy-underlined element and then select **View issues**.
+-->
 
 
 ### Issues are automatically ordered by severity
 
-Within each category of issues, first the errors are listed, then warnings, and then tips.
+Within each category of issues, first the errors are listed, then warnings, and then tips:
 
 ![The Issues tool displays Performance issues sorted by severity](./index-images/ordered-by-severity.png)
 
+(The **Severity** dropdown list controls whether Warnings, Errors, Tips, and Info items are listed.)
```

Comment:
Why use parenthesis here, I'm curious.

Reply:
@captainbrosset - influenced by lookahead that found that this point is focused on in a later section; the present mention of this point is therefore an aside, which can possibly use parentheses.  When I proceed to update the dedicated section below, about that point, I could remove parentheses here.  

Context: was figuring out and wondering why tips weren't listed; to match UI shown in live png, I had to do an extra undoc'd step of selecting Tips checkbox here.

Reply:
@captainbrosset - changed to: 

```
The **Severity** dropdown list controls whether Warnings, Errors, Tips, and Info items are listed.  See [Filter by severity](#filter-by-severity) below.
```